server/controllers/stock/reports/stock/lots_report.js

```diff
@@ -60,14 +61,16 @@ function stockLotsReport(req, res, next) {
     'enterprisePurchaseInterval', 'exhausted', 'expired',
     'inventory_uuid', 'lifetime_lot', 'min_delay',
     'min_months_security_stock', 'mvt_quantity', 'near_expiration',
-    'purchase_interval', 'tag_name', 'tracking_consumption',
-    'tracking_expiration', 'wac'];
+    'purchase_interval', 'tag_name', 'tracking_consumption', 'wac'];
 
   const dateKeys = ['min_stock_date', 'max_stock_date'];
 
   return Stock.getLotsDepot(null, options)
     .then((rows) => {
       rows.forEach(row => {
+        const current = new Date();
+        const delay = moment(new Date(row.expiration_date)).diff(current);
```

Comment:
What is the difference between this `delay` and the one that is included in the `purgeKeys`?  Can we just use that one instead of rewriting the function?  

I worry that if we change the delay logic in the future that these reports will get out of sync.

Reply:
The delay found in pureKeys is different because it is a decimal.